pages/workflow.md

```diff
@@ -1,54 +1,107 @@
+{% include nav.html %} {% include css.html %}
```

Comment:
Overall: fantastic workflow page! This is what I have been wanting to see in a workflow page, very expansive, very informative. Great!